README.md

```diff
@@ -202,6 +267,16 @@ For more advanced configuration, see the [W3C spec](http://www.w3.org/TR/cors/)
 
 JSONP is [not secure or performant](https://gist.github.com/tmcw/6244497). If IE8 or IE9 must be supported, use Microsoft's [XDomainRequest](http://blogs.msdn.com/b/ieinternals/archive/2010/05/13/xdomainrequest-restrictions-limitations-and-workarounds.aspx?Redirected=true) object instead of JSONP. There are [libraries](https://github.com/mapbox/corslite) to help with this.
 
+###SOAP Web Services
+*Todo: best practices for SOAP web services*
+
```

Comment:
Looks like there are a couple gaps.  Also, you should consider creating an index (hyperlinked) menu at the top to all people to jump around.  A lot of content in a scrolling format.